build/FindLameMSVC.cmake

```diff
@@ -10,11 +10,11 @@ endif(MINGW)
 message(STATUS ${LAMEMSVC_LIBRARY})
 
 if (LAMEMSVC_INCLUDE_DIR AND LAMEMSVC_LIBRARY)
-      set(LAMEMSVC_FOUND TRUE)
+      set(LAME_FOUND TRUE)
 endif (LAMEMSVC_INCLUDE_DIR AND LAMEMSVC_LIBRARY)
 
-if (LAMEMSVC_FOUND)
+if (LAME_FOUND)
       message (STATUS "Found vorbis: ${LAMEMSVC_LIBRARY}")
-else (LAMEMSVC_FOUND)
+else (LAME_FOUND)
       message (FATAL_ERROR "Could not find: vorbis")
```

Comment:
rather `message (FATAL_ERROR "Could not find: lame")`

Reply:
fixed